### PR TITLE
ALFREDOPS-330: first iteration.

### PR DESCRIPTION
**Pull request by anghelutar**
Amp which "overwrites" the default log4j.properties .

### CODE REVIEW

**Comment by anghelutar** [comment]
@tgeens @kerkhofsd Do you see any problem in merging this?

File: alfred-json-logging/src/main/amp/config/alfresco/extension/alfred-json-logging-log4j.properties [hunk 1]
```diff
@@ -0,0 +1,268 @@
...
+
+###### File appender definition #######
+log4j.appender.File=org.apache.log4j.DailyRollingFileAppender
+log4j.appender.File.File=alfresco.log
+log4j.appender.File.Append=true
+log4j.appender.File.DatePattern='.'yyyy-MM-dd
+log4j.appender.File.layout=org.apache.log4j.PatternLayout
+log4j.appender.File.layout.ConversionPattern=%d{yyyy-MM-dd} %d{ABSOLUTE} %-5p [%c] [%t] %m%n
+
+###### Hibernate specific appender definition #######
```
Comment (kerkhofsd) [inline alfred-json-logging/src/main/amp/config/alfresco/extension/alfred-json-logging-log4j.properties]: Are all following properties the Alfresco defaults? Or do we overwrite them by intention? 

If we can just use the default properties -> can't we just remove all of the following?

File: build.gradle [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+    version = "0.0.1"
+
+    repositories {
+        mavenLocal()
+        jcenter()
+    }
+}
+
+wrapper {
+    distributionType = Wrapper.DistributionType.ALL
```
Comment (kerkhofsd) [inline build.gradle]: wrapper version doesn't match executable version. 

The way @tgeens  advised: just remove this wrapper configuration & update executables with `./gradlew wrapper --gradleVersion = 5.6.1`

File: alfred-json-logging/src/main/amp/config/alfresco/extension/alfred-json-logging-log4j.properties (hunk 1 above)
Comment (anghelutar) [reply alfred-json-logging/src/main/amp/config/alfresco/extension/alfred-json-logging-log4j.properties]: We can keep only the appender layout, but then it does not work.

File: build.gradle (hunk 2 above)
Comment (tgeens) [reply build.gradle]: 5.6.2 is current :-)

**Pull request merged**